### PR TITLE
feat(web): Web Release Action

### LINKED ISSUE #14: Release GH actions

### PR DESCRIPTION
**Pull request by RayRedGoose**
## Issue
Resolve #14 

## Summary
Contains Release Action to force release job for `canvas-tokens-web` package.

## Release Category
Web Infrastructure

### CODE REVIEW

File: scripts/utils/update-changelog.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---`;
+
+const [prefix] = PACKAGE.split('-').reverse();
+const regex = new RegExp(`^# (${prefix}|all|components)`, 'i');
+
+const changelogBody = CHANGESET_BODY.split('##')
+  .filter(block => regex.test(block))
+  .map(b =>
+    b.replace(/# [a-zA-Z0-9_ ]*\n/g, a => {
+      const updatedTitle = a.replace(/# |\n/g, '').replace('Components', 'Other');
```
Comment (alanbsmith) [inline scripts/utils/update-changelog.ts:17]: Just adding some context. 😄 
```suggestion
      // Canvas Kit's changelog generator defaults section headings to "Components"
      // We're updating that default with use "Other" instead.
      const updatedTitle = a.replace(/# |\n/g, '').replace('Components', 'Other');
```

File: .github/workflows/release-web.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          git add . && git commit -m "chore: Release @workday/canvas-tokens-web v${{steps.new-tag.outputs.tag}} [skip release]" && git tag -a @workday/canvas-tokens-web@${{steps.new-tag.outputs.tag}} -m "@workday/canvas-tokens-web@${{steps.new-tag.outputs.tag}}"
+
+      - name: See git log
+        run: |
+          git log --no-walk --tags --oneline -n 1
+      
+      - name: Build
+        run: npm run build:tokens
+
+      # Push both the commit and tag created by Lerna's version command using a PAT
```
Comment (RayRedGoose) [inline .github/workflows/release-web.yml:105]:
```suggestion
      # Push both the commit and tag created by changeset version command using a PAT
```

File: .github/workflows/release-web.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+      # Push both the commit and tag created by changeset version command using a PAT
+      - name: Push changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GH_RW_TOKEN }}
+          branch: "main"
+          tags: true
+
+      ## Create a release on Github. This will trigger an internal Slack message
```
Comment (RayRedGoose) [inline .github/workflows/release-web.yml:113]:
```suggestion
      ## Create a release on Github.
```

**Review by alanbsmith — APPROVED** [review]

File: .github/workflows/release-web.yml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        with:
+          tag_name: "@workday/canvas-tokens-web@${{steps.new-tag.outputs.tag}}"
+          name: "@workday/canvas-tokens-web@${{steps.new-tag.outputs.tag}}"
+          body: ${{steps.changeset.outputs.body}}
+          draft: false
+          prerelease: false
+
+      # Publish to npm. Must be run after a build
+      - name: Publish
+        run: npx changeset publish
```
Comment (alanbsmith) [inline .github/workflows/release-web.yml:127]:
```
run: npx changeset publish --otp=${{ secrets.NPM_CI_PUBLISH_TOKEN}}
```

**Pull request merged**